ci/ansible/database.yml

```diff
@@ -0,0 +1,48 @@
+- hosts: databases
+  become: yes
+
+  vars:
+    postgresql_global_config_options:
+      - option: listen_addresses
+        value: '*'
+    postgresql_databases:
+      - name: "{{dbname}}"
+        owner: "{{dbuser}}"
+    postgresql_users:
+      - name: "{{dbuser}}"
+        db: "{{dbname}}"
+        password: "{{dbpass}}"
+        role_attr_flags: CREATEDB,CREATEROLE
+        priv: ALL
+    postgresql_auth_method: scram-sha-256
+    postgresql_hba_entries:
+      - {type: local, database: all, user: postgres, auth_method: peer}
+      - {type: local, database: all, user: all, auth_method: peer}
+      - {type: host, database: all, user: all, address: '127.0.0.1/32', auth_method: "{{ postgresql_auth_method }}"}
+      - {type: host, database: all, user: all, address: '::1/128', auth_method: "{{ postgresql_auth_method }}"}
+      - {type: host, database: "{{dbname}}", user: "{{dbuser}}", address: '10.0.0.0/24', auth_method: "{{ postgresql_auth_method }}"}
```

Comment:
Just realized the address was supposed to be a variable, will update when I get a chance